src/components/countries/country.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { BiChevronRightCircle } from 'react-icons/bi';
+import countryName from './countries_name';
+import './scss/country.scss';
+
+const Country = ({
+  country, totalConfirmed, handleClick,
+}) => {
+  const navigate = useNavigate();
+  const HandleRoute = (route) => {
+    navigate(route);
+  };
+  const isoName = countryName[country];
+
+  const mapImg = isoName ? `https://raw.githubusercontent.com/djaiss/mapsicon/33ba28808f8d32b5bae0ffada9cadd07073852e1/all/${isoName.toLowerCase()}/vector.svg` : '/World_map.png';
+  return (
+    <>
+      <button
+        className="country"
+        type="button"
+        onClick={() => {
+          HandleRoute('/country');
+          handleClick(country);
+        }}
+      >
+        <BiChevronRightCircle size={20} className="info-icon" color="white" />
+        <img className="country-map" alt="map" src={mapImg} width="100" height="100" />
```

Comment:
- [ ] Some data that is present in the desktop version is not visible in the mobile version. Remember, that this project is a mobile application and the mobile has to be in very good shape. Let's fix this error so that your app looks and performs as expected 💯. See the error below:

Your desktop version | Your mobile version
:---:|:---:
![](https://i.imgur.com/LuwxltF.png) | ![](https://i.imgur.com/8wg5Jef.png)